docs/architecture/cloud-native/toc.yml

```diff
@@ -1,16 +1,14 @@
 - name: "Architecting Cloud Native .NET Apps for Azure"
   href: index.md
   items:
-  - name: Introduction to Cloud Native Applications
-    href: introduction.md
+  - name: Introduction to Cloud-Native Applications
+    href: introducing-cloud-native.md
     items:
     - name: Defining Cloud Native
-      href: definition.md
+      href: defining-cloud-native.md
     - name: Candidate Apps for Cloud Native
       href: candidate-apps.md
-    - name: The Azure Cloud Native Development Stack
-      href: azure-development-stack.md
-  - name: Scaling Cloud Native .NET Applications
+    - name: Scaling Cloud Native .NET Applications
```

Comment:
there something a bit broken with this file because when I click on Introduction, it opens the Scaling article. Might be indentation here.

Also, with the change introduced last May for accessibility that a node cannot open and be clickable at the same time, you need to think if you really want the > nodes to repeat or if you want to give a different name like Chapter 1, Chapter 2, etc. and then have the content inside.

Reply:
FIxed

Reply:
This file is still broken. It's not a valid yml